### PR TITLE
Build changelog for 0.3.0 release

### PR DESCRIPTION
**Pull request by joyvelasquez**
Updated the XRTpy changelog files to hold the newest file for version three release.

### CODE REVIEW

File: docs/changelog/index.rst [hunk 1]
```diff
@@ -13,3 +13,4 @@ including bug fixes and changes to the application programming interface
 
    dev
    0.2.0
+   0.3.0
```
Comment (namurphy) [inline docs/changelog/index.rst:16]:
```suggestion
   0.3.0
   0.2.0
```
(putting newest ones first)

File: docs/changelog/0.3.0.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+XRTpy v0.3.0 (2023-03-31)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- Modified ``xrt_teem``` to return a namedtuple of SunPy maps. (:pr:`148`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:7]:
```suggestion
- Modified ``xrt_teem``` to return a `~collections.namedtuple` of SunPy maps. (:pr:`148`)
```

File: docs/changelog/0.3.0.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- New routine ``xrt_deconvolve``, uses the Hinode XRT point spread function and the Richardson-Lucy algorithm to deconvolve (sharpen) an XRT image. (:pr:`145`)
+- New functionality, ``xrt_remove_lightleak`` for subtracting light leak (visible stray light) image from XRT synoptic composite images. (:pr:`151`)
+
+Improved Documentation
+----------------------
+
+- Updated Python requirements for packages used in XRTpy.(:pr:`139,137`)
+- Redevelop the XRTpy GitHub issue templates site. (:pr:`154`)
+- Updated the bibliography and xrt_teem example notebook. (:pr:`158`)
+- Updated the XRT contam on ccd geny file, date up till 2023-05-20. (:pr:`160`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:21]:
```suggestion
- Updated the XRT contamination data file, which is now up-to-date through 2023-05-20. (:pr:`160`)
```

File: docs/changelog/0.3.0.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+--------
+
+- New routine ``xrt_deconvolve``, uses the Hinode XRT point spread function and the Richardson-Lucy algorithm to deconvolve (sharpen) an XRT image. (:pr:`145`)
+- New functionality, ``xrt_remove_lightleak`` for subtracting light leak (visible stray light) image from XRT synoptic composite images. (:pr:`151`)
+
+Improved Documentation
+----------------------
+
+- Updated Python requirements for packages used in XRTpy.(:pr:`139,137`)
+- Redevelop the XRTpy GitHub issue templates site. (:pr:`154`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:19]:
```suggestion
- Redeveloped the XRTpy GitHub issue templates. (:pr:`154`)
```

File: docs/changelog/0.3.0.rst [hunk 5]
```diff
@@ -0,0 +1,21 @@
+XRTpy v0.3.0 (2023-03-31)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- Modified ``xrt_teem``` to return a `~collections.namedtuple` of SunPy maps. (:pr:`148`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:7]:
```suggestion
- Modified `~xrtpy.response.xrt_teem.xrt_teem` to return a `~collections.namedtuple` of SunPy maps. (:pr:`148`)
```

File: docs/changelog/0.3.0.rst [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+Features
+--------
+
+- New routine ``xrt_deconvolve``, uses the Hinode XRT point spread function and the Richardson-Lucy algorithm to deconvolve (sharpen) an XRT image. (:pr:`145`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:12]:
```suggestion
- New routine `~xrtpy.util.xrt_deconvolve.xrt_deconvolve`, uses the Hinode XRT point spread function and the Richardson-Lucy algorithm to deconvolve (sharpen) an XRT image. (:pr:`145`)
```

File: docs/changelog/0.3.0.rst (hunk 5 above)
Comment (joyvelasquez) [reply docs/changelog/0.3.0.rst:7]: Awesome, I did not know this method create a reference link.

File: docs/changelog/0.3.0.rst [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+Features
+--------
+
+- New routine `~xrtpy.util.xrt_deconvolve.xrt_deconvolve`, uses the Hinode XRT point spread function and the Richardson-Lucy algorithm to deconvolve (sharpen) an XRT image. (:pr:`145`)
+- New functionality, `~xrtpy.util.xrt_remove_lightleak.xrt_remove_lightleak` for subtracting light leak (visible stray light) image from XRT synoptic composite images. (:pr:`151`)
+
+Improved Documentation
+----------------------
+
+- Updated Python requirements for packages used in XRTpy.(:pr:`139,137`)
```
Comment (namurphy) [inline docs/changelog/0.3.0.rst:18]:
```suggestion
- Updated Python requirements for packages used in XRTpy. (:pr:`139,137`)
```

**Pull request merged**